docs/contributing/style_guide.md

```diff
@@ -85,22 +85,17 @@ When designing function interfaces, stick to the following rules.
    there is only one, and it seems inconceivable that we would add more optional
    parameters in the future.
 
-<!-- prettier-ignore-start -->
-<!-- see https://github.com/prettier/prettier/issues/3679 -->
-
 3. The 'options' argument is the only argument that is a regular 'Object'.
 
    Other arguments can be objects, but they must be distinguishable from a
    'plain' Object runtime, by having either:
 
-    - a distinguishing prototype (e.g. `Array`, `Map`, `Date`, `class MyThing`)
-    - a well-known symbol property (e.g. an iterable with `Symbol.iterator`).
+   - a distinguishing prototype (e.g. `Array`, `Map`, `Date`, `class MyThing`)
+   - a well-known symbol property (e.g. an iterable with `Symbol.iterator`).
```

Comment:
I think this is ok? This should create a nested list, but maybe that depends on the renderer.